### PR TITLE
Express-post-JSON | Max

### PR DESCRIPTION
**Pull request by max-cendalski**
express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+app.post('/api/grades', (req, res) => {
+  res.setHeader('content-type', 'application/json', 'charset Utf-8');
+  var gradeObject = req.body;
+  gradeObject.id = nextId;
+  grades[nextId] = gradeObject;
+  nextId++;
+  res.json(grades);
+  res.status(201);
```
Comment (thebearingedge) [inline express-post-json/index.js:24]: Be sure to set the status code before sending the body. Otherwise, the custom 201 gets ignored.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 1 above)
Comment (max-cendalski) [reply express-post-json/index.js]: I fixed it

File: express-post-json/index.js [hunk 2]
```diff
@@ -20,8 +20,8 @@ app.post('/api/grades', (req, res) => {
   gradeObject.id = nextId;
   grades[nextId] = gradeObject;
   nextId++;
-  res.json(grades);
   res.status(201);
+  res.json(grades);
```
Comment (thebearingedge) [inline express-post-json/index.js:24]: Ok, good and like in the GIF, only the one object of the created grade should be in the response body.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 2 above)
Comment (max-cendalski) [reply express-post-json/index.js]: ok, I done

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**